profiler/src/Demos/Samples.Computer01/QuicklyDeadThreads.cs

```diff
@@ -0,0 +1,93 @@
+// <copyright file="QuicklyDeadThreads.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Samples.Computer01
+{
+    public class QuicklyDeadThreads
+    {
+        private const int SleepDurationMs = 0;
+        private ManualResetEvent _stopEvent;
+        private List<Task> _activeTasks;
+
+        private const int MaxCreatedThreadCount = 1024;
+
+        public QuicklyDeadThreads()
+        {
+        }
+
+        public void Start()
+        {
+            if (_stopEvent != null)
+            {
+                throw new InvalidOperationException("Already running...");
+            }
+
+            _stopEvent = new ManualResetEvent(false);
+            _activeTasks = new List<Task>
+            {
+                Task.Factory.StartNew(
+                    () =>
+                    {
+                        if (string.IsNullOrEmpty(Thread.CurrentThread.Name))
+                        {
+                            Thread.CurrentThread.Name = "QuickDeadThreads";
+                        }
+
+                        while (!_stopEvent.WaitOne(SleepDurationMs))
+                        {
+                            CreateQuicklyDeadThreads();
+                        }
+                    },
+                    TaskCreationOptions.LongRunning),
+            };
```

Comment:
why having a list of tasks (`_activeTasks`) for only one task?

Reply:
This is due to copy/paste of existing scenario: I noted to refactor them all and build specific base classes to handle the difference cases